translations.yml

```diff
@@ -272,6 +287,7 @@
   pt-BR: 'Fork e Editar no Github'
   fa: 'انشعاب گرفتن و ویرایش در گیت‌هاب'
   id: 'Fork &amp; Edit di Github'
+  pl: 'Rozwidl i Edytuj na Github'
```

Comment:
I am not sure if anyone will understand 'Rozwidl'; I would be keen to write 'Otwórz na Github', which encompasses any action, really.

Reply:
hmm, This is the correct form of the imperative mode of the polish  verb "rozwidlić".  "Open on GitHub" is not the same as to create a fork. You could use the expanded expression "Utwórz rozwidlenie... ," but this is worse.

Reply:
This is a tricky one, indeed. I think that 'Otwórz na GitHub' works best as it doesn't suggest the action to take beforehand, so the user can decide. The problem I have with 'Rozwidl' is that no one says it like this in reference to GitHub; everyone just uses 'Fork-it' in Poland. Or at least I haven't heard anyone saying 'Rozwidl'. We can't, however, write 'Fork-it na GitHubie', so I think 'Otwórz...' or 'Odwiedź' or 'Zobacz na GitHubie' would work better.

Reply:
Or will we be brave? How about 'forkuj'? Since we say "klonuj", "skroluj", "kopiuj"? Since you can find "forkowanie", "forkować" on the Internet? Let it be then:
"Forkuj i edytuj na Github".

Reply:
Wohoo! I love it!

Reply:
OK:
`pl:  'Forkuj i edytuj na Github'`

Reply:
Thank you both!